python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java

```diff
@@ -549,5 +557,23 @@ private void addSymbolUsage(Name nameTree) {
         symbol.addUsage(nameTree, Usage.Kind.OTHER);
       }
     }
+
+    private void addTypeToSymbol(Name nameTree, Expression rhs) {
+      Type type = null;
+      if (rhs.is(Kind.CALL_EXPR)) {
+        Expression callee = ((CallExpression) rhs).callee();
+        if (callee.is(Kind.NAME)) {
```

Comment:
Does that mean we won't be able to handle type resolution if the callee is a qualified expression instead? From the tests it seems so, but I don't understand why?

Reply:
For the time being yes, as a first baby step. I prefer to handle that case in a separate PR (and I documented with a unit test).

Reply:
fair enough! lgtm then